spec/loggability/log_devices/datadog_spec.rb

```diff
@@ -0,0 +1,39 @@
+# -*- rspec -*-
+#encoding: utf-8
+
+require 'rspec'
+
+require 'loggability/logger'
+require 'loggability/log_device/datadog'
+
+
+describe Loggability::LogDevice::Datadog do
+
+	let( :logger ) { described_class.new( 'datadog_api_key' ) }
+
+
+	it "The logger is an instance of Loggability::LogDevice::Datadog" do
+		expect( logger ).to be_instance_of( Loggability::LogDevice::Datadog )
+	end
+
+
+	it "It queses up log entries using the ::Thread::Queue data structure " do
+		expect( logger.logs_queue ).to be_instance_of( Thread::Queue )
+	end
+
+
+	it "verify that the http client is setup correctly" do
```

Comment:
I don't think we want to test for specific constant values. IMO, Configurations and constants should be mutable, and the tests should not make assumptions about what the values are. Having this tests just means we have to modify constants in two places instead of one.

Reply:
Just to elaborate, I think the tests below that e.g. the configured batch size is respected when posting to datadog, is valuable, but testing exactly the exact batch size value isn't.

Reply:
I put these specs in there to prevent someone from changing these values to above the datadog's payload limits and not realizing it. But I understand your point. I will just put a note on those constants.